frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenu.tsx

```diff
@@ -329,23 +347,34 @@ export const FeatureStrategyMenu = ({
                 )}
             </Dialog>
             {selectedTemplate && (
-                <LegacyReleasePlanReviewDialog
-                    open={addReleasePlanOpen}
-                    setOpen={(open) => {
-                        setAddReleasePlanOpen(open);
-                        if (!open) {
-                            setIsStrategyMenuDialogOpen(true);
-                        }
-                    }}
-                    onConfirm={() => {
-                        addReleasePlan(selectedTemplate);
-                    }}
-                    template={selectedTemplate}
-                    projectId={projectId}
-                    featureName={featureId}
-                    environment={environmentId}
-                    crProtected={crProtected}
-                />
+                <>
+                    <LegacyReleasePlanReviewDialog
+                        open={addReleasePlanOpen}
+                        setOpen={(open) => {
+                            setAddReleasePlanOpen(open);
+                            if (!open) {
+                                setIsStrategyMenuDialogOpen(true);
+                            }
+                        }}
+                        onConfirm={() => {
+                            addReleasePlan(selectedTemplate);
+                        }}
+                        template={selectedTemplate}
+                        projectId={projectId}
+                        featureName={featureId}
+                        environment={environmentId}
+                        crProtected={crProtected}
+                    />
+                    <ReleasePlanConfirmationDialog
+                        template={selectedTemplate}
+                        crProtected={crProtected}
+                        open={addReleasePlanConfirmationOpen}
+                        setOpen={setAddReleasePlanConfirmationOpen}
+                        onConfirm={() => {
+                            addReleasePlan(selectedTemplate, true);
+                        }}
+                    />
```

Comment:
cc @FredrikOseberg @kwasniew related to our conversation about being able to have more release plans in the future. Currently, we'd be showing a warning that you're about to replace. Doesn't change the existing behavior, so it shouldn't impact what you're doing, just keeping you posted ;)